scripts/build-manifest.js

```diff
@@ -0,0 +1,26 @@
+const manifest = require('../src/manifest.json');
+const version = require('../package.json').version;
+
+manifest.version = version;
+
+if (process.env.BROWSER === 'chrome') {
```

Comment:
These items are the differences I observed in the 2 manifests.

Reply:
I think that is the two items. @benjaminbhollon would know more as they did the original port.

I included the link in the ReadMe file as it was from mozilla (Firefox) in case anything was updated by them.